docs/challenges/how-challenges-work.mdx

```diff
@@ -26,10 +26,10 @@ Once you've selected a language, we'll create the git repository for you and pro
   You need to run `git` commands on your local machine.
   <Steps>
     <Step title="Open Terminal on your system">
-      If you're using Windows, make sure to [install WSL](https://learn.microsoft.com/en-us/windows/wsl/install).
+      On Windows, you can use [Command Prompt `(cmd)`](https://www.youtube.com/results?search_query=how+to+open+cmd) or [PowerShell](https://www.youtube.com/results?search_query=how+to+open+powershell), but we recommend using [WSL](https://learn.microsoft.com/en-us/windows/wsl/install).
     </Step>
     <Step title="Set up Git">
-      Make sure Git is [set up properly](https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup).
+      Make sure Git is [installed](https://git-scm.com/download/win) and [set up properly](https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup).
```

Comment:
Again, this isn't a windows specific guide - we can't link to the windows download link.  How about https://git-scm.com/downloads

Reply:
Will do.